### DISCUSSION TITLE
How to calculate the Negative electrode current-driven interface utilisation factor [m3.mol-1]?

### DISCUSSION THREAD

**Discussion by redchang111** [pybamm-team/PyBaMM#3400] [General]
I want to simulate the degradation of LG Chem cylindrical INR18650 MJ1 lithium-ion battery, but I don't know their Negative electrode current-driven interface utilisation factor [m3.mol-1], someone can tell me how to calculate it?

**Comment by brosaplanella** [comment]
The utilisation factor is computed by fitting to experimental data.
